docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,189 @@ Authorization
-see all active contracts*").
+Your Daml application sends requests to the :doc:`Ledger API </app-dev/ledger-api>` exposed by a participant node to submit changes to the ledger
+(e.g., "*exercise choice X on contract Y as party Alice*"), or to read data from the ledger
+(e.g., "*read all active contracts visible to party Alice*").
+Your application might send these requests via a middleware like the :doc:`JSON API </json-api/index>`.
 
-What requests are valid is defined by :ref:`integrity <da-model-integrity>` and :ref:`privacy <da-model-privacy>` parts the :ref:`Daml Ledger Model <da-ledgers>`.
-This model is defined in terms of :ref:`Daml parties <glossary-party>`,
-and does not require any cryptographic information to be sent along with requests.
+Whether a participant node *can* serve such a request depends on whether the participant node hosts the respective parties, and
+whether the request is valid according to the :ref:`Daml Ledger Model <da-ledgers>`.
+Whether a participant node *will* serve such a request to a Daml application depends on whether the application accompanied the
+request with an access token that is valid and sufficient to authorize the request for this participant node.
 
-In particular, this model does not talk about authentication ("*Is the request claiming to come from Alice really sent by Alice?*")
-and authorization ("*Is Alice authorized to add a new Daml package to the ledger?*").
+Acquiring and using access tokens
+*********************************
 
-In practice, Daml ledgers will therefore need to add authentication to the ledger API.
-
-.. note::
-    Depending on the ledger topology, a Daml ledger may consist of multiple participant nodes,
-    each having its own ledger API server.
-    Each participant node typically hosts different Daml parties,
-    and only sees data visible to the parties hosted on that node (as defined by the Daml privacy model).
-
-    For more details on Daml ledger topologies, refer to the :ref:`Daml Ledger Topologies <daml-ledger-topologies>` documentation.
-
-Adding authentication
-=====================
-
-How authentication is set up on a particular ledger is defined by the ledger operator.
-However, most authentication setups share the following pattern:
+How an application should acquire access tokens depends on the participant node it talks to and is ultimately setup by the participant node operator.
+However most setups share the following pattern:
 
 First, the Daml application contacts a token issuer to get an access token.
-The token issuer verifies the identity of the requesting user
-(e.g., by checking the username/password credentials sent with the request),
-looks up the privileges of the user,
+The token issuer verifies the identity of the requesting application
+(e.g., by checking the `OAuth 2.0 client credentials <https://datatracker.ietf.org/doc/html/rfc6749#section-2.2>`_ sent with the request),
```

Comment:
Does the person reading the documentation need to know how it's done, or only that it's done?

Reply:
Addressed.